src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletController.java

```diff
@@ -33,4 +30,9 @@ public ResponseEntity<WalletDTO> createWallet(@Valid @RequestBody CreateWalletDT
 
         return new ResponseEntity<>(walletDTO, headers, HttpStatus.CREATED);
     }
+    @GetMapping(value = "/{id}")
+    public ResponseEntity<WalletDTO> findWalletById(@RequestParam long id) {
```

Comment:
I suggest to use `@PathVariable` annotation instead of `@RequestParam`. For now, the example endpoint for id = 1 which has to be executed for get method looks like the follow: 

`http://localhost:8080/api/wallet/{id}?id=1`